docs/user-guide/ldap-configuration.md

```diff
@@ -74,23 +99,31 @@ Sync settings options include how often everything syncs: Full sync interval (se
 
 After filling in all the required information, click the <button className="button button--primary button-iom">**Create**</button> button.
 
-## LDAP actions
+### LDAP actions
 
 After creating LDAP, you will see action buttons on the left side below the input fields.
-<Img src="/img/user-guide/ldap/ldap-actions.png" alt="LDAP actions (Sync all users, Sync changed users, Remove imported users, Delete LDAP)"/>
+<Img src="/img/user-guide/ldap/ldap-action-new.png" alt="LDAP actions (Sync all users, Sync changed users, Remove imported users, Delete LDAP)"/>
 
-### Sync all users
+#### **Sync All Users**
```

Comment:
not sure if we can something about this, but it would have been nice if this showed in the left side menu as well if I'm quickly looking for something:

![CleanShot 2024-10-08 at 21 04 16](https://github.com/user-attachments/assets/747b3eea-9abd-46ec-bd7b-e5b316554bf8)